app-misc/dconf/dconf-0.40.0.recipe

```diff
@@ -0,0 +1,98 @@
+SUMMARY="A simple key/value storage for reading"
+DESCRIPTION="dconf is a simple key/value storage system \
+that is heavily optimised for reading. \
+This makes it an ideal system for storing user preferences \
+(which are read 1000s of times for each time the user changes one). \
+It was created with this usecase in mind."
+HOMEPAGE="https://gitlab.gnome.org/GNOME/dconf/"
+COPYRIGHT="2010 Ryan Lortie
+	2010 Tomasz Miąsko"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://gitlab.gnome.org/GNOME/dconf/-/archive/$portVersion/dconf-$portVersion.zip"
+CHECKSUM_SHA256="f83e7e993ddca6d0e29ffa659a8b0acce80093cae0e24a60405e5427dbb4009b"
+SOURCE_DIR="$portVersionedName"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="?x86"
+
+libVersionCompat=1.0.0
```

Comment:
Check this for libVersion/libVersionCompat: https://github.com/haikuports/haikuports/blob/4f180bdb94db9749ae800820b271e262a12754aa/media-libs/openexr/openexr-2.2.1.recipe#L22